Dockerfile

```diff
@@ -30,6 +30,17 @@ RUN  \
 	rm $GORELEASER_DOWNLOAD_FILE && \
 	goreleaser -v
 
+# install cosign
+ARG COSIGN_VERSION=1.2.1
```

Comment:
what about using something like this?
https://docs.docker.com/develop/develop-images/multistage-build/#use-an-external-image-as-a-stage

Reply:
copy --from=gcr.io/projectsigstore/cosign:v1.2.1  /bin/cosign /usr/local/bin/cosign

https://github.com/philips-labs/spiffe-vault/blob/465654d738696af5585175d185dcf8e4399c2432/example/spiffe-vault-cosign/Dockerfile#L1

cc: @cpanato

Reply:
done